resources/protos/commit.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto2";
+
+package tutorial;
+
+option java_package = "app";
+option java_outer_classname = "CommitProtos";
+
+message Flag {
+    string type;
+    string value;
+}
+
+message Commit {
```

Comment:
Can you document all protobuf messages and all their fields that are not obvious? Protobuf files are read often.